pkg/lang/ir/compile.go

```diff
@@ -204,7 +204,7 @@ func (g Graph) GetEntrypoint(buildContextDir string) ([]string, error) {
 	}
 
 	template := `set -e
-/var/envd/bin/envd-sshd --authorized-keys %s --port %d --shell %s &
+/var/envd/bin/envd-ssh --authorized-keys %s --port %d --shell %s &
```

Comment:
I think we should rename here too.

Reply:
I've tried to rename `var/envd/bin/envd-ssh` to `var/envd/bin/envd-sshd` here, but it could't pass e2e tests, more info: https://github.com/tensorchord/envd/actions/runs/3027354180.

Reply:
That's because the base image now does not contain sshd. Maybe we can use sshd and make a new release to see if it works.

Reply:
OK, I'll fix it.

Reply:
Shall we checkout a new branch for release test?

Reply:
Sure. I created a new branch dev for the test.

Reply:
Could you please open the PR to this branch `dev`?

Reply:
Sure, see https://github.com/tensorchord/envd/pull/866

Reply:
https://github.com/tensorchord/envd/releases/tag/v0.2.0-alpha.20 Let's see if it works.

Reply:
It works, let's create a new PR with the dev branch, does it work for you?

Reply:
I've run e2e tests on `dev` branch with `v0.2.0-alpha.20` release: 

- make e2e-cli-test
```shell
Ran 12 of 12 Specs in 163.797 seconds
SUCCESS! -- 12 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestMain (163.80s)
PASS
coverage: 40.8% of statements in ./pkg/app
ok      github.com/tensorchord/envd/e2e/cli     163.878s        coverage: 40.8% of statements in ./pkg/app
```
- make e2e-lang-test
```shell
Ran 5 of 5 Specs in 438.409 seconds
SUCCESS! -- 5 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestMain (438.41s)
PASS
coverage: 21.5% of statements in ./pkg/app
ok      github.com/tensorchord/envd/e2e/language     438.712s     coverage: 21.5% of statements in ./pkg/app
```

It seems that it works as expected.

Now, maybe we could re-run the github ci checks.

Reply:
> It works.
> 
> Does the PR have the same change as the merged one?

Yes, https://github.com/tensorchord/envd/pull/864 and https://github.com/tensorchord/envd/pull/866 have the same change.